server/plugin/api.go

```diff
@@ -970,27 +938,24 @@ func (p *Plugin) createIssueComment(c *UserContext, w http.ResponseWriter, r *ht
 	p.writeJSON(w, result)
 }
 
-func (p *Plugin) getYourAssignmentsData(c *UserContext) []*github.Issue {
-	config := p.getConfiguration()
-
-	githubClient := p.githubConnectUser(c.Context.Ctx, c.GHInfo)
+func (p *Plugin) getLHSData(c *UserContext) ([]*github.Issue, []*github.Issue, []*github.Issue) {
```

Comment:
should we return here a list of `[]*github.Issue` instead of having the same parameters multiple times?